provenance/chains-config.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
does the current config automatically pick this up or does it have to be "applied" some how on the cluster when updated?

Reply:
the gcb step should automatically apply this everytime here:

https://github.com/GoogleContainerTools/distroless/blob/main/cloudbuild.yaml#L111

Reply:
sick